MFE/devices/src/view/devices/View.jsx

```diff
@@ -164,6 +181,10 @@ const Devices = () => {
     setSearchText(search);
   };
 
+  const handleCloseMultipleDevicesDeletionErrorAlert = () => {
+    setMultipleDevicesDeletion({ ...multipleDevicesDeletion, showing: false, devices: [] });
```

Comment:
You can remove the `...multipleDevicesDeletion`, since you are setting all values after it.